<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="matrixTable">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>20</y>
      <width>541</width>
      <height>501</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>130</y>
      <width>191</width>
      <height>271</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="generateButton">
       <property name="text">
        <string>Сгенерировать матрицу</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="sortButton">
       <property name="text">
        <string>Отсортировать матрицу</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="obnulButton">
       <property name="text">
        <string>Обнулить матрицу</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="sizeSpinBox">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>30</number>
       </property>
       <property name="value">
        <number>5</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
